subprojects/packagefiles/yaml-cpp/meson.build

```diff
@@ -0,0 +1,74 @@
+project(
+  'yaml-cpp',
+  'cpp',
+  version: '0.8.0',
+  meson_version: '>=0.63.0',
+  default_options: ['cpp_std=c++11'],
+  license: 'MIT',
+)
+
+inc = include_directories('include')
+subdir('src')
+yaml_cpp_private_inc = include_directories('include', 'src')
+
+project_version_split = meson.project_version().split('.')
+lib_cpp_args = []
+cc = meson.get_compiler('cpp')
+if cc.get_argument_syntax() == 'gcc'
```

Comment:
You could get rid of this check and just check for supported arguments (here and in the `test` directory), which is a good thing to do anyway.